api/internal/datasource/config.go

```diff
@@ -12,5 +17,13 @@ func NewConfigFromEnv() (*Config, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if cfg.RemoteMLFlowBaseUrl == "" {
+		return nil, errors.New("CDSW_API_URL is required")
+	}
+	if cfg.RemoteProjectID == "" {
+		return nil, errors.New("CDSW_PROJECT_NUM is required")
```

Comment:
the variable says ID but the env var is NUM. I think both exist. Which do we want? Either the variable or env var and this message prob need to be renamed

Reply:
Oh yeah inconsistent errors, will fix

Reply:
Fixed.